3.sc_Morphology_Linear_Modeling/scripts/2b.Linear_Modeling_Visualization.r

```diff
@@ -66,18 +69,16 @@ lm_df$channel_learned <- dplyr::recode(lm_df$channel_cleaned,
     )
 
 print(dim(lm_df))
-head(lm_df)
+head(lm_df,2)
 unique(lm_df$channel_learned)
 lm_df$abs_Metadata_number_of_singlecells <- abs(lm_df$Metadata_number_of_singlecells)
 lm_df$abs_twob_Metadata_Treatment_Inhibitor_Dose <- abs(lm_df$twob_Metadata_Treatment_Inhibitor_Dose)
 lm_df$abs_Treatment_Dose <- abs(lm_df$Treatment_Dose)
 
-head(lm_df)
-
 loop_list <- unique(lm_df$inducer1_inhibitor_inhibitor_dose__inducer1_dose)
 x_list <- c('abs_twob_Metadata_Treatment_Inhibitor_Dose','abs_Treatment_Dose')
 
-pdf(file=lm_cp_fig )
+pdf(file=lm_cp_fig_abs )
```

Comment:
Consider adding some documentation here.

Reply:
I might remove this in the future, I am holding off on extra changes for now